src/main/java/io/archivesunleashed/io/ArchiveRecordWritable.java

```diff
@@ -15,11 +15,6 @@
  * limitations under the License.
  */
 
- /**
```

Comment:
ArchivesRecordWritable is not deprecated.  Instead GenericArchivesRecordWritable was deprecated and is replaced with this.  The "Generic" was removed because it does not need to be distinguished from Arc & Warc.  (That's why the double spacing in the comments as I searched and replaced the old file.)

Reply:
The deprecation message was clear that the GenericArchivesRecordWritable would be replaced with ArchivesRecordWritable in the future release.  In future, it would be better to add the new file while deprecating the old one, I agree.  Mea Culpa.

Reply:
If necessary, I could restore GenericArchivesRecordWritable for a second round of messaging.